manifests/server.pp

```diff
@@ -423,6 +423,33 @@
     require => Class['zabbix::repo'],
     tag     => 'zabbix',
   }
+  
+  # Ensure that the correct config file is used.
+  
+  if $::osfamily == 'debian' {
+		file { '/etc/init.d/zabbix-server':
+			ensure  => present,
+			mode    => '0755',
+			require => Package["zabbix-server-${db}"],
+			replace => true,
```

Comment:
So, you mean to leave only the _content_ attribute so it always builds the file from the template ?